sdk/core/Azure.Core/src/Shared/OperationInternalBase.cs

```diff
@@ -27,7 +27,7 @@ protected OperationInternalBase(ClientDiagnostics clientDiagnostics, Response ra
             _updateStatusScopeName = $"{operationTypeName}.UpdateStatus";
             _scopeAttributes = scopeAttributes?.ToDictionary(kvp => kvp.Key, kvp => kvp.Value);
             RawResponse = rawResponse;
-            DefaultPollingInterval = TimeSpan.FromSeconds(1);
+            PollingStrategy = new ConstantPollingStrategy();
```

Comment:
We should check if Retry-After is in the rawResponse if so instantiate a RetryAfterStrategy.  If there is ever a case where some of the responses will have Retry-After but others won't then RetryAfterStrategy can take in a fallback, but this shouldn't be needed.